### DISCUSSION TITLE
Allows users to change the configuration of sw360 from the GUI

### DISCUSSION THREAD

**Discussion by hoangnt2** [eclipse-sw360/sw360-frontend#269] [Ideas]
Currently, the configuration of sw360 is located in **/etc/sw360/sw360.properties**. Users cannot change the configuration from GUI.

**Proposal**:
Create a new GUI where the user can change the configuration from

**Comment by hoangnt2** [comment]
@heliocastro, I have created a discussion following https://github.com/eclipse-sw360/sw360-frontend/pull/264#issuecomment-2122563053.
@deo002, @amritkv, how do you think about this idea?

**Reply by amritkv** [reply]
Good to have feature !
